RecoParticleFlow/PFProducer/src/PFCandConnector.cc

```diff
@@ -158,7 +158,8 @@ void PFCandConnector::analyseNuclearWPrim(PFCandidateCollection& pfCand,
 
   math::XYZTLorentzVectorD momentumSec;
```

Comment:
I think momentumSec must be initialized, to cover the case when the `if` below is not satisfied.
Alternatively, an `else` after line 162

Reply:
I did check that the default initialization is (0,0,0,0), as desired, but true that it's better to be explicit.

Reply:
> I did check that the default initialization is (0,0,0,0), as desired, but true that it's better to be explicit.

@kdlong are you planning to initialize momentumSec?

Reply:
Sorry @clacaputo, it's done now. I also tried to protect against divide by zero in a few other places where it hypothetically could happen. The code would clearly benefit from cleanup but I didn't want to be too ambitious